examples/background_task.py

```diff
@@ -37,6 +38,32 @@ async def time_task(self):
     async def before_my_task(self):
         await self.wait_until_ready()  # Wait until the bot logs in
 
+    # Schedule every 10s; each run takes between 5 to 20s. With overlap=2, at most 2 runs
+    # execute concurrently so we don't build an ever-growing backlog.
+    @tasks.loop(seconds=10, overlap=2)
+    async def fetch_status_task(self):
+        """
+        Practical overlap use-case:
+
+        Poll an external service and post a short summary. Each poll may take
+        between 5 to 20s due to network latency or rate limits, but we want fresh data
+        every 10s. Allowing a small amount of overlap avoids drifting schedules
+        without opening the floodgates to unlimited concurrency.
+        """
+        print(f"[status] start run #{self.fetch_status_task.current_loop}")
```

Comment:
Attributing to a separate variable was correct because of overlap, just the name felt weird